database/mysql/mysql_user.py

```diff
@@ -206,31 +218,81 @@ def connect(module, login_user=None, login_password=None, config_file=''):
     db_connection = MySQLdb.connect(**config)
     return db_connection.cursor()
 
+# User Authentication Management was change in MySQL 5.7
+# This is a generic check for if the server version is grater than 5.6
+def server_version_check(cursor):
+    cursor.execute("SELECT VERSION()");
+    result = cursor.fetchone()
+    version_str = result[0]
+    version = version_str.split('.')
+
+    if (int(version[0]) <= 5 and int(version[1]) < 7):
```

Comment:
@morgo I believe this is the line of code you are referring to. This logic does not include 5.7 as a `True` result. It will return `False` which is the expected behavior. I will fix the typo in the comment. Also, the comment is inverted since it returns `True` when the version is 5.6 or lower. Very sorry for the confusion.